example-proxy.ini

```diff
@@ -65,4 +63,18 @@ attribute = sAMAccountName
 # The timeout is specified in seconds.
 # This feature is EXPERIMENTAL.
 enabled = false
-timeout = 3
\ No newline at end of file
+timeout = 3
+
+[preamble-cache]
+enabled = true
+timeout = 5
+attribute = objectclass
+value-prefix = App-
+
+[realm-mapping]
+strategy = static
+# Realm to use for authentication (may also be left blank)
+realm =
+# strategy = preamble
```

Comment:
Exactly! i.e. the mapping strategy that uses the "preamble cache" for lookup. And we could easily implement other realm mapping strategies.